docs/Legal/Customer Agreement.mdx

```diff
@@ -0,0 +1,85 @@
+# Acuerdo de Servicio
```

Comment:
Isn't the pricing section the "Pago" section. Maybe I should change the name. Or should it include the specific figures to be paid and the recurrency?

Reply:
The terms and conditions shall be included in a contract form and then and only then signed by both parties. Unlike other services open to the public this is a tailored solution and the terms should be agreed upon the subscription to the service

Reply:
Yes, I think you should change the name and include thes pecific figures to be paid and the recurrency. I think pricing translates mora as "Precios" than "Pago". With regard to the second point, I think it is perfect, although it should be specified so that we do not appear to have forgotten.

Reply:
Okay. I have reestructured the first section to clear that confussion and I am going to specify the pricing as well